cheatsheets/gleam-for-python-users.md

```diff
@@ -765,6 +793,7 @@ two different types they must be wrapped in a new custom type.
 from typing import Union
 
 OptionalString = Union[str, None]
+OptionalString = str | None  # newer syntax
```

Comment:
Let's remove the old syntax

Reply:
Removed the old syntax and also updated the Python example to be similar to the Gleam example.